src/util/address.rs

```diff
@@ -370,6 +370,7 @@ impl Display for Address {
                 let mut prefixed = [0; 21];
                 prefixed[0] = match self.network {
                     Network::Bitcoin => 0,
+                    Network::Signet => 125,
```

Comment:
Changes in this file verifiable here: https://github.com/kallewoof/bitcoin/blob/2b9db9ac0936cf278ad27c59497d9feec61f6440/src/chainparams.cpp#L334-L340